data-transfer-service/src/main/java/com/epam/pipeline/dts/transfer/service/impl/DtsSynchronizationService.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017-2021 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.dts.transfer.service.impl;
```

Comment:
It seems like a good idea to move this class to `com.epam.pipeline.dts.sync.service` package.

Reply:
Done, moved all new classes to `com.epam.pipeline.dts.sync` and subpackages inside